cmd/influxd/run.go

```diff
@@ -90,6 +90,9 @@ func Run(config *Config, join, version string, logWriter *os.File) (*messaging.B
 			go a.ListenAndServe()
 		}
 
+		// broker needs the server to occasionally run continuous queries
+		b.RunContinuousQueryLoop(s)
```

Comment:
How can you be sure `b` will be non-nil? If you look at line 46 it's checking if b is non-nil. There will be no broker object on a data-only node, right?